pom.xml

```diff
@@ -285,8 +286,9 @@
               <goal>build</goal>
             </goals>
             <configuration>
+              <outputDirectory>${project.build.directory}/native/org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}</outputDirectory>
```

Comment:
This isn't going to work for Android though. We need to place the libraries in `${javacpp.platform.library.path}` for that platform. We should be able to use the profiles below in the pom.xml file for that though. Could you give this a try?

Reply:
Should be enough, let's see what that gives, thanks!

`javacpp.platform.library.path` isn't available for any other platform. If neither it nor `outputDirectory` is set, the effective output directory gets based on the scanned classes, so we can't really set something like `javacpp.platform.library.path` before actually parsing everything in those cases.